### PR TITLE
add a question to javascript file

### PR DESCRIPTION
**Pull request by ddoyediran**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

**Comment by ddoyediran** [comment]
Please, let me know if there is anything to add or remove. Thank you @jdwilkin4

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -744,7 +795,20 @@ const javascriptQuiz = [
     Distractor3: ".trim()",
     Explanation:
       "The .includes() method is used to check if one string is found in another.",
-    Link: "https://www.freecodecamp.org/news/javascript-string-contains-how-to-use-js-includes/",
+    Link:
+      "https://www.freecodecamp.org/news/javascript-string-contains-how-to-use-js-includes/",
+  },
+  {
+    Question:
+      "In JavaScript, variables that are accessible throughout the program are called?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:803]:
```suggestion
      "In JavaScript, what is the name for variables that are accessible throughout the program?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -744,7 +795,20 @@ const javascriptQuiz = [
...
+    Answer: "Global variables",
+    Distractor1: "Const variables",
+    Distractor2: "Local variables",
+    Distractor3: "No variable is accessible throughout in JavaScript",
+    Explanation:
+      "Global variables are declared for accessibility throughout the program",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:809]:
```suggestion
      "Global variables are declared and made accessible throughout the entire JavaScript program.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by ddoyediran** [comment]
Thank you! Done

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**